core/src/test/scala/unit/kafka/server/DynamicBrokerConfigTest.scala

```diff
@@ -327,10 +327,12 @@ class DynamicBrokerConfigTest {
     EasyMock.replay(kafkaServer)
 
     props.put(KafkaConfig.ListenersProp, "PLAINTEXT://hostname:9092,SASL_PLAINTEXT://hostname:9093")
-    val newConfig = KafkaConfig(props)
+    new DynamicListenerConfig(kafkaServer).validateReconfiguration(KafkaConfig(props))
 
+    // it is illegal to update configs of existent listeners
+    props.put("listener.name.plaintext.you.should.not.pass", "failure")
```

Comment:
It is possible to update configs of existing listeners if they are dynamic, right?

Reply:
you are right. I will revise the comment.